### ISSUE TITLE
Albums appear multiple times in the search results.

### ISSUE THREAD

**Issue by ManfredHi** [gerbera/gerbera#2878]
If I search for the name of an artist in BubbleUPNP or Hifi-Cast, for example, the albums of this artist are displayed several times in the search results. Namely, just as often as they are in the virtual folders.
In my case I get the same thing 5 times because the album is in the year folder, in the genre, in the artist, in the album artist and in the general albums.

What do I have to set so that the albums found are only displayed once?

**Comment by KarlStraussberger** [comment]
This can be solved easily, if you have a fresh database (not migrated from 1.1). You have to set `searchable-container-flag="yes"` in `upnp` config section. See https://docs.gerbera.io/en/latest/config-server.html#upnp

If you have an old database, you have to reimport to set the `searchable` flag only once in virtual layout.

**Comment by ManfredHi** [comment]
Thanks for the info, but somehow that doesn't work.
I'm starting with an empty database on 1.12.1.
The “searcheble-container-flag” is set to “yes” for me. If I set "const chain = { album: {searchable: true ....} in the common.js file, then the album appears multiple times in the search. In my case 5 times because the album is in 5 different virtual folders occurs.
If I set searchable: false, then the album is not searched for at all.
Then I only get title search results.

Thanks for the info, but somehow that doesn't work.
I'm starting with an empty database on 1.12.1.
The “searcheble-container-flag” is set to “yes” for me. If I set "const chain = { album: {searchable: true ....} in the common.js file, then the album appears multiple times in the search. In my case 5 times because the album is in 5 different virtual folders occurs.
If I set searchable: false, then the album is not searched for at all.
Then I only get title search results.

Sorry for my bad english.
